test/boulder-tools/build.sh

```diff
@@ -36,11 +36,11 @@ go get \
 
 wait
 
-# protoc-gen-go outputs a line that says:
-# const _ = grpc.SupportPackageIsVersion4
-# so it will fail to compile with a different version of the grpc package.
-# Since we currently have version 4 of the grpc package vendored, we have to
-# build a specific version of protoc-gen-go.
+# grpc uses a version attestation variable of the form grpc.SupportPackageIsVersionN
+# where N is the minor release version of the protoc-gen-go version used to
+# generate the protobuf mappings that are used. So that we don't have to worry
```

Comment:
Rather than starting with "So that..." I think this sentence would read better if you start with explaining what we do, then why, e.g. "Do X so that ..."

Also, instead of "not worrying," I'd say that the reason we check this out is that we can still reproduce a correct version that matches our vendored libary even if protoc-gen-go bumps this version number.